qualification/conftest.py

```diff
@@ -470,9 +472,56 @@ async def cbf(
             await pcc.request("capture-stop", name)
 
 
+class CoreAllocator:
+    """Provide CPU cores to receivers that need them.
+
+    It is initialised with a list of cores (from pytest config), with earlier
+    entries considered better than later ones. Cores are allocated in this
+    order. There is no mechanism to return cores; simplify create a new
+    allocator to start fresh.
```

Comment:
*simply create a new allocator to start fresh.

Reply:
Thanks for catching.